### DISCUSSION TITLE
benefits is not available about 4days

### DISCUSSION THREAD

**Discussion by brain305** [community/community#135802] [GitHub Education]
### Hi everyone,

I reqeusted student account in aug 12, and it approved. but my github account couldn't use copilot. please activate my account free github pro.

**Comment by davevad93** [comment]
Hi @brain305 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @brain305 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by prabhjotschugh** [comment]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!
